aksel.nav.no/website/app/(routes)/(produktbloggen)/produktbloggen/_ui/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { BLOGG_LANDINGSSIDE_BLOGS_QUERYResult } from "@/app/_sanity/query-types";
+import { humanizeRedaksjonType } from "@/ui-utils/format-text";
+import { Avatar, avatarUrl } from "../../../../dev/_ui/avatar/Avatar";
+
+type Blogg =
+  NonNullable<BLOGG_LANDINGSSIDE_BLOGS_QUERYResult>["bloggposts"][number];
+
+export const queryToAvatars = (
+  queryResponseEditorialStaff: Blogg["writers"],
+): Avatar[] => {
+  return (
+    queryResponseEditorialStaff?.map((queryData) => ({
```

Comment:
Instead off falling back in `""` everywhere, we can just filter "invalid" entries out beforehand.